components/nats-init/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM alpine:3.11
+LABEL source=git@github.com:kyma-project/kyma.git
+
+WORKDIR /nats-streaming/
+
+COPY scripts/prepare-config.sh .
```

Comment:
Why don't we need the licences?
```
COPY licenses ./licenses
```

Reply:
which licenses would you copy in?

Reply:
The licences which are pulled [here](https://github.com/kyma-project/kyma/blob/master/common/makefiles/generic-make-go.mk#L84).

Reply:
`docker.mk` does not pull any licenses. This target is only valid in `generic-make-go.mk`

Reply:
The question is don't we need the licenses anymore? Cause we had licenses in the previous image of nats-init.

Reply:
Please feel free to ignore my comments if needed. I don't wanna block this PR. It is just not clear to me why we don't need the licenses anymore.

Reply:
it was empty in the previous images as well

Reply:
so, i checked our license puller script https://github.com/kyma-project/test-infra/blob/master/prow/scripts/license-puller.sh
there is just no code that would pull a license for a bash only image.